reverse_image_search_module/search_image.py

```diff
@@ -81,6 +83,55 @@ def find_index_from_image(img, n):
     return idx, dist
 
 
+def find_repeating_index(idx, dist):
+    counter = {}
+
+    for ide, distance in zip(idx, dist):
+        if counter.get(ide, None) is None:
+            record = [distance, 1]
+        else:
+            record = counter[ide]
+            record[0] += 1.0 /distance
```

Comment:
Distance here is the cosine similarity, right? Between -1 and 1

Reply:
I remove the function :) it was ussless at the end